docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -97,87 +100,103 @@ The zkApp CLI will ask if you would like to create an acompnanying ui project. S
 LICENSE           build             jest.config.js    package-lock.json tsconfig.json
 README.md         config.json       keys              package.json
 babel.config.cjs  jest-resolver.cjs node_modules      src
 ```
 
-We will be working mostly within the `src` directory, which will contain the TypeScript code for our smart contract. When we build or deploy (which automatically builds for us), our TypeScript will be compiled into JavaScript inside the `build` directory.
+For this tutorial, you work mostly in the `src` directory that contains the TypeScript code for the smart contract. Each time you make updates, then build or deploy (which automatically builds), the TypeScript code is compiled into JavaScript in the `build` directory.
 
 ### Preparing the project
 
-We will start by creating files for our project and deleting the default files that come with the new project.
+Start by deleting the default files that come with the new project.
 
-First, delete the old files. Run:
+To delete the old files:
 
 ```sh
 $ rm src/Add.ts
 $ rm src/Add.test.ts
 $ rm src/interact.ts
 ```
 
-And generate the new files for our project. Enter:
+Then, generate the new files for the project:
 
 ```sh
 $ zk file src/Square
 $ touch src/main.ts
 ```
 
-The `zk file <name>` command created both `src/Square.ts` and `src/Square.test.ts` for us, but we won't worry about writing tests in this tutorial. We'll use `main.ts` as a script to interact with the smart contract and observe how it works. In later tutorials, we will see how to interact with a smart contract from the browser, like a typical end user, but for now we'll just use our `main.ts` script to do this.
+The `zk file` command created the `src/Square.ts` and `src/Square.test.ts` test files. However, this tutorial does not include writing tests. Instead, you use `main.ts` as a script to interact with the smart contract and observe how it works. 
 
-Now, let's open `src/index.ts` in a text editor and change it to look like below. This file contains all exports we want to make available for consumption from outside our smart contract project, such as from a UI.
+In later tutorials, you learn how to interact with a smart contract from the browser, like a typical end user. For now, you just use the `main.ts` script.
+
+Now, open `src/index.ts` in a text editor and change it to look like:
 
 ```ts src/index.ts
 1 import { Square } from './Square.js';
 2
 3 export { Square };
 ```
 
+The `src/index.ts` file contains all of the exports you want to make available for consumption from outside our smart contract project, such as from a UI.
+
 ### Building and running
 
 :::info
 
-The following commands are included for reference, and will fail if you run them at this point. This is because we haven't written the `Square` smart contract yet.
+The following commands are included for reference and will fail if you run them now. This is expected behavior because you haven't written the `Square` smart contract yet.
```

Comment:
I'm not clear why we include these commands for reference when we know they will fail... talk me through a scenario where adding the commands here is useful

Maybe we need more visibility that these are commands we know will succeed after the tuto is complete?